src/main/webapp/components/LabelList.vue

```diff
@@ -0,0 +1,89 @@
+<template>
+  <div class="labels-list">
+    <!-- School Label -->
+    <v-chip
+      class="ma-1"
+      color="orange"
+      dark label
+      :close="canModify"
+      close-icon='mdi-find-replace'
```

Comment:
nit: I think it's more readable to close tags on the same line as the last line in the tag. Same goes for the tags below.
```suggestion
    close-icon='mdi-find-replace'>
```

Reply:
+1

Reply:
Sounds good. I was following the [examples on Vuetify's website](https://codepen.io/pen/?&editable=true&editors=101=https%3A%2F%2Fnext.vuetifyjs.com%2Fen%2Fcomponents%2Falerts%2F) but I think I agree. This also seems to align with [Google's HTML guidelines](https://google.github.io/styleguide/htmlcssguide.html#HTML_Style_Rules).

I'll make those changes.

Reply:
LGTM with style changes :)